Kitodo/src/main/java/org/kitodo/production/forms/LdapServerForm.java

```diff
@@ -121,6 +131,27 @@ public void setLdapServerById(int ldapServerID) {
         }
     }
 
+    /**
+     * Gets decrypted manager password.
+     *
+     * @return The decrypted manager password
+     */
+    public String getDecryptedManagerPassword() {
+        if (AESUtil.isEncrypted(ldapServer.getManagerPassword())) {
+            String securitySecret = ConfigCore.getParameterOrDefaultValue(ParameterCore.SECURITY_SECRET);
+
+            if (StringUtils.isNotBlank(securitySecret)) {
+                try {
+                    ldapServer.setManagerPassword(AESUtil.decrypt(ldapServer.getManagerPassword(), securitySecret));
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
Using `e.printStackTrace()` is not good. Log this exception if this is needed and you want to handle this exception here.